lms/djangoapps/instructor_task/docs/decisions/002-processing-scheduled-instructor-tasks.rst

```diff
@@ -0,0 +1,31 @@
+=====================================
+Processing Scheduled Instructor Tasks
+=====================================
+
+Status
+------
+
+Draft
+
+Background
+----------
+The previous ADR (``Scheduled Instructor Tasks``) explains the motivation and approach for scheduling instructor tasks. 
+
+This ADR will cover our approach for the execution of scheduled instructor tasks.
+
+Decision
+--------
+
+A management command will handle submitting scheduled tasks to Celery that are due for execution. This management command will rely on new utility functions introduced to the **instructor_task** app to query tasks that require processing.
+
+This management command will be invoked by a Jenkins job running on a (cron) schedule to process scheduled tasks due for execution.	
```

Comment:
Can we write down information about the frequency of the management command? Does it run once per day? Once per hour?
The frequency is import to understand the expectation developers should have on the due_date column of the InstructorTask Schedule model.

Reply:
Well, provide an example frequency would be nice. And you can talk about how frequency will affect the `due` field.